src/org/thoughtcrime/securesms/database/DatabaseFactory.java

```diff
@@ -1416,11 +1417,15 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         }
       }
 
-      if (oldVersion < INTRODUCED_PINNED_MESSAGES){
+      if (oldVersion < INTRODUCED_PINNED_MESSAGES) {
         db.execSQL("ALTER TABLE sms ADD COLUMN pinned BOOLEAN DEFAULT 0;");
         db.execSQL("ALTER TABLE mms ADD COLUMN pinned BOOLEAN DEFAULT 0;");
       }
 
+      if (oldVersion < INTRODUCED_NICK_NAMES) {
+        db.execSQL("ALTER TABLE recipient_preferences ADD COLUMN nick_name TEXT DEFAULT 'null';");
```

Comment:
Is this the `"null"` string or `NULL` ?

Reply:
I had a hard time setting db attr to NULL using update statement since they always set it to "NULL" so i decided to make the default "null"  since java knows null but not NULL for some reason,